### PR TITLE
Added clarifications to README.md

### PR DESCRIPTION
**Pull request by marczwalua**
Added some instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,23 @@
 [![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)][godoc]
 [![Travis CI](https://travis-ci.org/secrethub/secrethub-go.svg?branch=master)][travis-ci]
 [![GolangCI](https://golangci.com/badges/github.com/secrethub/secrethub-go.svg)][golang-ci]
-[![Go Report Card](https://goreportcard.com/badge/github.com/secrethub/secrethub-go)][goreportcard]
+[![Go Report Card](https://goreportcard.com/badge/github.com/secrethub/secrethub-go)][goreportcard] ![Licence](https://img.shields.io/hexpm/l/plug.svg)
```
Comment (SimonBarendse) [inline README.md]: Please add this on a new line.

File: README.md [hunk 2]
```diff
@@ -3,15 +3,23 @@
...
+
+
```
Comment (SimonBarendse) [inline README.md]: Please remove redundant newlines.

File: README.md [hunk 3]
```diff
@@ -3,15 +3,23 @@
...
+> SecretHub is a developer tool to help you keep database passwords, API tokens, and other secrets out of IT automation scripts.
 
 <img src="https://secrethub.io/img/secrethub-gopher.png" alt="Gopher" width="200px"/>
 
-## Installation
+## Getting started
+
+### Prerequisites
+
+In order to use the Go client library, you need a __SecretHub account__. You can do this by following the first instructions [here](https://secrethub.io/docs/getting-started/) (*Installing the CLI* and *signing up for a SecretHub account*).
```
Comment (SimonBarendse) [inline README.md]: What do you think of:

In order to use the SecretHub Go client library, you need an account on SecretHub. You can create a free account by [signing up through the CLI](https://secrethub.io/docs/getting-started/).

?

Comment (SimonBarendse) [inline README.md]: In theory, one could also sign up using the client itself, but probably most users will use the CLI first and then move to using the client.

Comment (mackenbach) [reply README.md]: Yes make it that.

Comment (marczwalua) [reply README.md]: I agree on that.

**Comment by marczwalua** [comment]
@SimonBarendse I implemented your feedback on the README. Could you re-review?

**Pull request merged**